<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Sayı Tahmin Oyunu</string>
   </property>
   <widget class="QLabel" name="lbl1Aralik">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1. Aralık</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl2Aralik">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2. Aralık</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneditAralik1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneditAraik2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSayiUret">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tahmin Edilecek Sayıyı Üret</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTahminEt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tahmin Et</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textSonuc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
